src/components/Counter/__tests__/counter-snapshot-tests.jest.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import Counter from "../Counter";
+
+describe("Counter renders correctly", () => {
+  it("with count below the limit", () => {
+    const tree = renderer.create(<Counter count={1} />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it("with count above limit", () => {
+    const tree = renderer.create(<Counter count={1000} />).toJSON();
```

Comment:
If we will change the default numbers limit in the future, this test will break. Let's pass here also the maxDigits prop.

Reply:
you can delete this test (and change the name of the other test which check the count with limit to ""with count above limit")